homework-submission/react-week3/src/render-todo.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function RenderTodo(props) {
+  let styles = { textDecoration: props.todo.done ? "line-through" : "" }
```

Comment:
don't make any style in Js file
give it an className and identified  it in css file